recommender/migrate/utils.py

```diff
@@ -0,0 +1,156 @@
+# pylint: disable=invalid-name
+
+"""Util methods for migration purposes"""
+
+import os
+from importlib import import_module
+
+from definitions import MIGRATE_DIR
+from recommender.models.migration import Migration
+
+
+def _get_migration_class(migration: Migration):
+    migration_name = migration.name
+    capitalized = [part.capitalize() for part in migration_name.split("_")[1:]]
+    class_name = "".join(capitalized)
```

Comment:
If you require changing `underscore_case` to `CamelCase` or any other there are packages for that, it might be better at handling edge cases.